src/convertToObject.js

```diff
@@ -3,16 +3,27 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles (see an example in
+ * [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties (see an exampl in
+ * [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesObject = {};
+  const stylesArray = sourceString.split(';');
+
+  stylesArray.filter(style => style.includes(':')).forEach(element => {
+    const [key, value] = element.split(':');
+
+    stylesObject[key.trim()] = value.trim();
+  });
```

Comment:
1. I don't think that `filter` is an efficient solution here. I think you can check inside `forEach` if the key and value exists and then add it to object. The less loops the better.
2. 🚨Formatting: If you apply a chain of methods to an array each of them should start on new line
```
  stylesArray
    .filter(style => style.includes(':'))
    .forEach(element => {
      const [key, value] = element.split(':');

      stylesObject[key.trim()] = value.trim();
    });